Content.Server/Engineering/EntitySystems/SpawnAfterInteractSystem.cs

```diff
@@ -42,6 +43,9 @@ bool IsTileClear()
             if (!IsTileClear())
                 return;
 
+            if (tileRef.IsBlockedTurf(true))
+                return;
+
```

Comment:
should put this check into `IsTileClear` since that also runs on the postcheck for the doafter. that way if someone constructs a wall while you're placing the inflatable it wont place ontop